wallet/res/layout/dialog_tax_category_explainer.xml

```diff
@@ -0,0 +1,315 @@
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/reclassify_transactions_by_default_message"
+            android:textSize="13sp"
+            app:layout_constraintStart_toStartOf="@+id/incoming_title"
+            app:layout_constraintTop_toBottomOf="@+id/incoming_title" />
+
+        <TextView
+            android:id="@+id/income_text"
+            style="@style/Caption"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="5dp"
+            android:backgroundTint="@color/green_300"
+            tools:text="Income"
+            android:textColor="@color/green_300"
+            android:textSize="13sp"
+            app:layout_constraintBottom_toBottomOf="@+id/incoming_default"
+            app:layout_constraintStart_toEndOf="@+id/incoming_default"
+            app:layout_constraintTop_toTopOf="@+id/incoming_default" />
+
+        <TextView
+            android:id="@+id/you_can_change_it_two"
+            style="@style/Caption"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/reclassify_transactions_change_it_message"
+            android:textSize="13sp"
+            app:layout_constraintStart_toStartOf="@+id/incoming_default"
+            app:layout_constraintTop_toBottomOf="@+id/incoming_default" />
+
+        <TextView
+            android:id="@+id/transfer_in_text"
+            style="@style/Caption"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="5dp"
+            android:background="@color/steel_gray_40"
+            tools:text="Transfer-in"
+            android:textColor="@color/light_dark_gray"
+            android:textSize="13sp"
+            app:layout_constraintBottom_toBottomOf="@+id/you_can_change_it_two"
+            app:layout_constraintStart_toEndOf="@+id/you_can_change_it_two"
+            app:layout_constraintTop_toTopOf="@+id/you_can_change_it_two" />
+
+        <ImageView
+            android:id="@+id/left_right_arrow"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="20dp"
+            android:padding="8dp"
+            android:src="@drawable/ic_circle_left_right_arrow_gray"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/down_arrow"
+
+            />
+
+        <TextView
+            android:id="@+id/linked_account_title"
+            style="@style/Caption.SemiBold"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="10dp"
+            android:text="@string/reclassify_transactions_linked_accounts_title"
+            android:textSize="14sp"
```

Comment:
The design says 14px for the font size which lines up with Body2.  The design says weight of 600, which I don't see a correlation with our styles.

I can use "Body2.Medium" style and locally specify the color or should the "Body2" all use "@color/light_gray_black" instead of the "@color/gray_900"

Perhaps "@color/gray_900" was the original text color in the designs, but it has been updated to "@color/light_gray_black"

```xml
        <TextView
            android:id="@+id/linked_account_title"
            style="@style/Body2.Medium"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:layout_marginStart="10dp"
            android:text="@string/reclassify_transactions_linked_accounts_title"
            android:textColor="@color/light_gray_black"
            app:layout_constraintBottom_toBottomOf="@+id/left_right_arrow"
            app:layout_constraintStart_toEndOf="@+id/left_right_arrow"
            app:layout_constraintTop_toTopOf="@+id/left_right_arrow" />
```

Reply:
`gray_900` was our primary content color, but now it's `light_gray_black`. Although the naming of the latter can use some improvements which I'm doing in the CrowdNode branch.

What you have here is fine. Once we switch the font and I do some cleanup for the colors to match the new color system, I expect it'll be less confusing.